### PR TITLE
Refactor back-end part of review: small defects

### PR DESCRIPTION
**Pull request by DmytryiStriletskyi**
Pull request contains: create common namespace, squash migrations, fix typos.

### CODE REVIEW

File: acceptor/settings.py [hunk 1]
```diff
@@ -38,8 +38,8 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
-    'analytics.apps.AnalyticsConfig',
-    'charts.apps.ChartsConfig',
+    'edx_global_analytics.analytics.apps.AnalyticsConfig',
+    'edx_global_analytics.charts.apps.ChartsConfig',
```
Comment (z4y4ts) [inline acceptor/settings.py]: Why `edx_global_analytics`, but not `edx_olga`?

File: edx_global_analytics/analytics/test_fixture.json [hunk 2]
```diff
@@ -0,0 +1 @@
... 1, "active_students_amount_month": 1000, "active_students_amount_week": 1000, "students_per_country": "{\"RU\": 2632, \"CA\": 18543, \"UA\": 2011, \"null\": 1}", "data_created_datetime": "2016-02-25T20:24:26Z", "courses_amount": 116}}, {"pk": 124, "model": "analytics.edxinstallation", "fields": {"latitude": 1.0, "platform_name": "test", "platform_url": "test", "longitude": 1.0, "secret_token": "platform 0"}}, {"pk": 125, "model": "analytics.installationstatistics", "fields": {"active_students_amount_day": 2925, "edx_installation": 125, "statistics_level": 1, "active_students_amount_month": 1000, "active_students_amount_week": 1000, "students_per_country": "{\"RU\": 2632, \"CA\": 18543, \"UA\": 2011, \"null\": 1}", "data_created_datetime": "2016-02-26T20:24:26Z", "courses_amount": 117}}, {"pk": 125, "model": "analytics.edxinstallation", "fields": {"latitude": 1.0, "platform_name": "test", "platform_url": "test", "longitude": 1.0, "secret_token": "platform 0"}}, {"pk": 126, "model": "analytics.installationstatistics", "fields": {"active_students_amount_day": 2950, "edx_installation": 126, "statistics_level": 1, "active_students_amount_month": 1000, "active_students_amount_week": 100
```
Comment (z4y4ts) [inline edx_global_analytics/analytics/test_fixture.json]: Please, make it multiline.
And add trailing newline.

File: edx_global_analytics/analytics/utils.py [hunk 3]
```diff
@@ -1,11 +1,12 @@
 # pylint: disable-all
 
 """
-Utilities/
+Utilities, that creates test data for charts and map.
```
Comment (z4y4ts) [inline edx_global_analytics/analytics/utils.py]: creates -> create

File: acceptor/settings.py (hunk 1 above)
Comment (DmytryiStriletskyi) [inline acceptor/settings.py]: `OLGA` - OpenedX Learners Global Analytics. `edx_olga` seems tautological with `edx` at the start.
I think `edx_global_analytics` more obviously and has good combination with second `.analytics` and `.charts`.

File: edx_global_analytics/analytics/test_fixture.json (hunk 2 above)
Comment (DmytryiStriletskyi) [inline edx_global_analytics/analytics/test_fixture.json]: Look at [utilities file](https://github.com/raccoongang/acceptor/blob/4e1f4c316cc9f06a8e8d90787b20f430b84a2386/edx_global_analytics/analytics/utils.py#L5). 
There is `Needed to be deleted in production.` string.
Make it multiline is unnecessary move. Especially for **test fixture**.

File: edx_global_analytics/analytics/utils.py (hunk 3 above)
Comment (DmytryiStriletskyi) [inline edx_global_analytics/analytics/utils.py]: Thanks.

File: edx_global_analytics/analytics/test_fixture.json (hunk 2 above)
Comment (z4y4ts) [inline edx_global_analytics/analytics/test_fixture.json]: Fair enough. Then add a list what's to be deleted to the `utils.py`

File: acceptor/settings.py (hunk 1 above)
Comment (z4y4ts) [inline acceptor/settings.py]: What about `olga.analytics.apps.AnalyticsConfig`?
I'd like to keep names short, but reasonable

File: edx_global_analytics/analytics/test_fixture.json (hunk 2 above)
Comment (DmytryiStriletskyi) [inline edx_global_analytics/analytics/test_fixture.json]: Ok. Thanks.

File: acceptor/settings.py (hunk 1 above)
Comment (DmytryiStriletskyi) [inline acceptor/settings.py]: Fair enough. I`ll do it! %)

**Comment by z4y4ts** [comment]
Squash and merge it.

**Pull request merged**